src/screens/TypographyScreen.tsx

```diff
@@ -31,7 +33,12 @@ export const TypographyScreen = (): JSX.Element => {
             Investigate the issue about using `useCallback` on `onChange`
             https://github.com/adobe/react-spectrum/issues/2320
           */}
-          <Switch mx={4} my={8} value={colorMode === 'dark'} onChange={toggleColorMode} />
+          <Switch
+            mx={4}
+            my={8}
+            value={colorScheme === 'dark'}
+            onChange={() => setColorSchemeSetting(colorScheme === 'dark' ? 'light' : 'dark')}
```

Comment:
onChange nie działa na webie, ale jeśli zmienimy na onToggle to będzie działać i na webie i na mobilce